tests/integration/test_diagnostic_run.py

```diff
@@ -1,6 +1,7 @@
 """Test diagnostic script runs."""
```

Comment:
There should be no need to change the tests in this file, just delete `test_diagnostic_run_config_file` (using the `--config-file` command line argument no longer supported) and remove the skip marker from `test_diagnostic_run`.